kobuki_gazebo_plugins/src/gazebo_ros_kobuki_loads.cpp

```diff
@@ -309,6 +309,8 @@ void GazeboRosKobuki::setupRosApi(std::string& model_name)
 {
   std::string base_prefix;
   gazebo_ros_->node()->param("base_prefix", base_prefix, std::string("mobile_base"));
```

Comment:
You are right in case of the ROS parameter server, but this is not the case here. The parameter here comes from Gazebo and is set here: https://github.com/yujinrobot/kobuki_desktop/blob/indigo/kobuki_gazebo/launch/includes/robot.launch.xml#L9

If I provide an empty string, I believe the parameter is considered not set and the default is selected. At least this happend when I tried it. How did you make it work?